TF_Service_dotNet/TouchFree/Interactions/InteractionModules/AirPushInteraction.cs

```diff
@@ -91,6 +89,13 @@ public class AirPushInteraction : InteractionModule
             };
         }
 
+        protected override void OnInteractionSettingsUpdated(InteractionConfigInternal _config)
+        {
+            base.OnInteractionSettingsUpdated(_config);
+
+            thetaOne = ignoreDragging ? 15f : 65f;
```

Comment:
The narrower angle is set to 15 degrees, but looking at the previous default value above it was 22 degrees. Is this additional narrowing on purpose?